WooCommerce/Classes/ViewRelated/Dashboard/Settings/SettingsViewController.swift

```diff
@@ -58,6 +58,20 @@ final class SettingsViewController: UIViewController {
     ///
     private var sites = [Yosemite.Site]()
 
+    /// Payment Gateway Account ResultsController
+    ///
+    private let gatewayAccountResultsController: ResultsController<StoragePaymentGatewayAccount> = {
+        let storageManager = ServiceLocator.storageManager
+        /// TODO - resolve how I want to manage the siteID here. Should I just fetch all gatewayAccounts and filter by siteID later perhaps?
+        let predicate = NSPredicate(format: "siteID == %lld", siteID)
```

Comment:
Extra super mage nitpick comments, but siteID is probably an optional at this point, so maybe the results controller would hav to be optional as well?

Reply:
In https://github.com/woocommerce/woocommerce-ios/pull/4381/commits/775566b5dcc26310f255e389af0a617e923a744c I removed this change - I'm deferring view controller changes to the last pull in this series.